### ISSUE TITLE
Incorrect capitalization warning

### ISSUE THREAD

**Issue by InSyncWithFoo** [redhat-developer/lsp4ij#450]
I'm getting this warning:

![String 'languageServer.presentableName' is not properly capitalized. It should have title capitalization](https://github.com/user-attachments/assets/78350a2c-4f27-45c8-b342-ded25abbbfe1)

The warning is for the key name itself rather than the corresponding `.properties` entry, according to the quick fix.

Perhaps some specialized logic in Plugin DevKit is at play, since the same definitions in, for example, [`ConfigurableEP.java`](https://github.com/JetBrains/intellij-community/blob/ca1b5a7de414e13ff1ab59aa1a40858f2549f214/platform/ide-core/src/com/intellij/openapi/options/ConfigurableEP.java) work perfectly: keys and bundles are recognized as such and can be navigated to via the go-to-definition feature.

**Comment by angelozerr** [comment]
> Perhaps some specialized logic in Plugin DevKit is at play

Yes I think so, but I don't know if it is possible to contribute to teh DevKit plugin?

**Comment by angelozerr** [comment]
@InSyncWithFoo to be honest with you I will not have time to work on this issue because I prefer working on another issue like LSP API.
